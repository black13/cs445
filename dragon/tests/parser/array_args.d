program array_assign();
var a, b: array [12..4] of integer;
begin

end.
