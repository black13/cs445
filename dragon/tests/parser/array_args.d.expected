PROGRAM `array_assign` WITH ARGS:
AND VARIABLES:
  DECLARATIONS WITH TYPE ARRAY [`12` .. `4`] OF INTEGER:
    `a`
    `b`
AND SUBPROGRAMS:
DOES:
-- done dumping ast --
